### ISSUE TITLE
TDHttpClient produces java.lang.IncompatibleClassChangeError exception 

### ISSUE THREAD

**Issue by kalu44i** [treasure-data/td-client-java#20]
Hi Muga!

When I try to start a program which uses td-client 0.7.0 on Debian, it receives exception:
```
Exception in thread "2567" java.lang.IncompatibleClassChangeError: Implementing class
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:800)
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
	at java.net.URLClassLoader.defineClass(URLClassLoader.java:449)
	at java.net.URLClassLoader.access$100(URLClassLoader.java:71)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:361)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:355)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:354)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:425)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:308)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:358)
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:800)
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
	at java.net.URLClassLoader.defineClass(URLClassLoader.java:449)
	at java.net.URLClassLoader.access$100(URLClassLoader.java:71)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:361)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:355)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:354)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:425)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:308)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:358)
	at com.treasuredata.client.TDHttpClient.<init>(TDHttpClient.java:89)
	at com.treasuredata.client.TDClient.<init>(TDClient.java:126)
	at com.treasuredata.client.TDClientBuilder.build(TDClientBuilder.java:286)
	at com.metricinsights.collector.plugin.treasuredata.Runner.executeQuery(Runner.java:46)
	at com.metricinsights.collector.plugin.treasuredata.TreasureDataPlugin.executeQuery(TreasureDataPlugin.java:30)
	at com.metricinsights.collector.standalone.DataCollector.main(DataCollector.java:261)
``` 
Looks like it borns in the TDHttpClient class in 89 row
```
this.httpClient = config.useSSL ? new HttpClient(new SslContextFactory()) : new HttpClient();
```

If I use td-client 0.5.9, there isn't this exception.

Would you give some advice how to overcome this problem? Maybe you've met it before?

Thanks.

**Comment by xerial** [comment]
@kalu44i 
We haven't seen this error. I guess this error is caused by different versions of jetty client exist in your classpath. td-client-java 0.7.x uses HttpClient of jetty (version 9.2.2.v20140723). If your class path contains another version of jetty, this error could happen.

**Comment by muga** [comment]
can you please tell us what you've used your application server and the version?

**Comment by xerial** [comment]
@kalu44i 
We have just released td-client-0.7.2, and added a standalone jar file (td-client-0.7.2-jar-with-dependencies.jar), which probably solves your problem:
http://repo1.maven.org/maven2/com/treasuredata/client/td-client/0.7.2/

**Comment by kalu44i** [comment]
@muga @xerial 
I've found what was going on. One of our jar file used a lib which has jetty 8.2.2 which has a diff impl as 9.2.2 and greater. And our program always got it and thrown error.

Thanks for your help. 

Thanks you.
